### ISSUE TITLE
fade strategy lr

### ISSUE THREAD

**Issue by antoniskef** [XuyangBai/TransFusion#97]
hello, 
I would like to ask what learning rate and learning rate strategy I should use for the last 5 epochs without the db sampler. I have trained the transfusion_nusc_pillar_L for 15 epochs with cyclic strategy. Did I make a mistake and the cycle should be for 20 epochs and not for 15?

**Comment by guifentian** [comment]
Hi, did you solve the problem and got the same NDS as in the paper?

**Comment by antoniskef** [comment]
no, I started the training again
